### PR TITLE
Issue 49 user auth email verification

### PR DESCRIPTION
**Pull request by tiusty**
Adds email authentication when the user creates an account. Now users cannot login/user their account until they verify their account via through their email. This will prevent spam accounts and force users to use a real email address.

### CODE REVIEW

**Review by seanrayment — APPROVED** [review]
This looks good

**Review by akashkumar2296 — APPROVED** [review]

File: cocoon/scheduler/models.py [hunk 1]
```diff
@@ -38,7 +23,7 @@ class ItineraryModel(models.Model):
     itinerary = models.FileField(blank=True)
     agent = models.ForeignKey(MyUser, related_name='scheduled_tours', on_delete=models.SET_NULL, blank=True, null=True)
     tour_duration_seconds = models.IntegerField(default=0)
-    selected_start_time = models.OneToOneField('TimeModel', on_delete=models.SET_NULL, blank=True, null=True)
+    selected_start_time = models.DateTimeField(default=timezone.now, blank=True, null=True)
```
Comment (seanrayment) [inline cocoon/scheduler/models.py]: I think we want this to default to `None`

**Review by seanrayment — APPROVED** [review]
One comment but otherwise looks ready to merge to me

File: cocoon/scheduler/models.py (hunk 1 above)
Comment (tiusty) [reply cocoon/scheduler/models.py]: Lol good catch! yes it should definitely be none lol

**Pull request merged**